trade-finance-manager-ui/server/controllers/utilisation-reports/add-to-an-existing-payment/index.test.ts

```diff
@@ -84,6 +108,19 @@ describe('controllers/utilisation-reports/add-to-an-existing-payment', () => {
           },
         ],
       });
+      expect((res._getRenderData() as AddToAnExistingPaymentViewModel).availablePaymentsHeading).toEqual(
```

Comment:
Could we split the assertions regarding payment groups into a separate test? This now has a lot of assertions and so a lot of different ways to fail